### DISCUSSION TITLE
Help me ...

### DISCUSSION THREAD

**Discussion by YTLoudYN** [beeware/toga#1905] [Q&A] [answered]
![联想截图_20230426201042](https://user-images.githubusercontent.com/131869902/234570954-de58489e-b58f-464d-9c8c-fe29d114e23b.png)

How to achieve it by for  
![联想截图_20230426201120](https://user-images.githubusercontent.com/131869902/234571608-1a5e1faf-ebc1-4479-81e0-9c3b6aa8803f.png)

**Comment by freakboy3742** [comment]
I'd like to help, but I don't understand your question. What do you mean by "circulate"?

**Reply by YTLoudYN** [reply]
[on_press=self.on[How to give parameters]]

**Reply by proneon267** [reply, marked as answer]
> [on_press=self.on[How to give parameters]]

@YTLoudYN Use a lambda like: `on_press=lambda x: myFunction(arg1,args2)`

**Reply by YTLoudYN** [reply]
Thanks  I got it

**Reply by YTLoudYN** [reply]
![QQ截图20230429093225](https://user-images.githubusercontent.com/131869902/235277399-b8aa332f-dafc-4ac3-bdbd-3d009423b080.png)
![xx截图20230429093305](https://user-images.githubusercontent.com/131869902/235277403-d4eb7e96-b409-4990-99a2-7f61197c3104.png)
I implemented it with closures
